### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by kumarankit999**
## Fixes Issue
This PR Closes Issue #118 

## Changes proposed
I Have change something in Contributors  and Added Support Section.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,10 +83,17 @@ Envless is licensed under the GNU Affero General Public License v3.0 License - s
...
 + We also welcome you to join our [Slack](https://dub.sh/envless-slack) or [Discord](https://dub.sh/envless-discord) community for either support or contributing guidance.
 
 <a href="https://github.com/envless/envless/graphs/contributors">
   <p>
     <img src="https://contrib.rocks/image?repo=envless/envless" alt="A table of avatars from the project's contributors" />
   </p>
 </a>
+<h2>🙏 Support</h2>
+<p>
+Don't forget to leave a star ⭐️
```
Comment (dahal) [inline README.md:97]: Thanks for adding this, I would love to have a gif showing how to star, something like this (but full width) would be great 🌟 

Repo: https://github.com/calcom/cal.com#stay-up-to-date

**Review by dahal — CHANGES_REQUESTED** [review]
Also please make sure to tag the associated issue with key words like

Fixes #xxxx or Closes#xxxx so when this PR is merged, its issue will also be closed automatically.

File: README.md (hunk 1 above)
Comment (kumarankit999) [reply README.md:97]: I will be thankful, If you let me Know, How Can I do this one.

File: README.md [hunk 2]
```diff
@@ -83,10 +83,21 @@ Envless is licensed under the GNU Affero General Public License v3.0 License - s
...
+  </p>
+  
+  ![Enveless](https://user-images.githubusercontent.com/111564362/221513732-189c3917-2ef5-48f8-b5d9-334492389764.gif)
```
Comment (dahal) [inline README.md:100]: Thanks @kumarankit999 for quick turnaround. Please add this gif to ./github/images folder and link it from there instead. Here is an example

Image: https://github.com/envless/envless/blob/main/.github/images/discord-banner.png

**Review by dahal — CHANGES_REQUESTED** [review]

**Comment by kumarankit999** [comment]
Is adding files is disabled.

**Comment by money8203** [comment]
Hey Ankit, go to .github folder in your cloned repo, there you'll see an image folder. Inside the image folder add the gif and then copy the path from there -> paste it instead of the path you've put for the gif currently.

**Comment by kumarankit999** [comment]
Thanks,a lot Manish

**Comment by kumarankit999** [comment]
I have done it ,and created a PR

**Comment by money8203** [comment]
You don't have to create a new pr Ankit, instead you should merge it into this one only

**Comment by dahal** [comment]
@money8203 @kumarankit999 it would be nice if you can setup a pairing session to fix this. It would be a good learning experience for @kumarankit999

**Comment by kumarankit999** [comment]
OK that's great

**Review by money8203 — APPROVED** [review]
Looks good, thanks for you contribution Ankit 👍

**Review by dahal — APPROVED** [review]
Thanks guys, LGTM 🚀

**Pull request merged**